Amplitude/Amplitude.h

```diff
@@ -106,9 +105,7 @@
  */
 - (void)initializeApiKey:(NSString*) apiKey;
 - (void)initializeApiKey:(NSString*) apiKey userId:(NSString*) userId;
-- (void)initializeApiKey:(NSString*) apiKey userId:(NSString*) userId startSession:(BOOL)startSession;
 
-- (void)startSession;
```

Comment:
Should we mark these methods as deprecated instead of completely removing them? __attribute__((deprecated))